rc/tools/git.kak

```diff
@@ -247,7 +248,7 @@ define-command -params 1.. \
             fi
         fi
 
-        if [ "$wrapped" = true ]; then
+        if "$wrapped"; then
```

Comment:
Why change this? I'm not even sure this works as expected either.

Reply:
Previously it was comparing the string stored in $wrapped to the string "true".

Now it's executing the program named in $wrapped, which should effectively be `/bin/true` or `/bin/false`. It's safe because $wrapped is always set to one or the other now, it's not going to execute some random other thing.

It's probably less efficient to launch an external command than to do a string comparison, but it's not like this executed frequently.

Reply:
But why modify it? The hunk is not related to the changeset, it doesn't fix any issue, it's not a performance improvement, and it's harder (slightly) to understand what the new code does.

Reply:
I agree that this change should not really be here, reverted.

Correct me if I'm wrong, but I believe that this executes shell-builtins `true`/`false` if they exist (not guaranteed by POSIX). I find it more elegant than `[ $bool = true ]` or `test -n` but I guess it's not so common.

Reply:
I checked `dash(1)` and `bash(1)`; only dash lists `true` as a shell builtin (as a synonym for `:`), and bash lists neither.

Now that I look more closely, `command -V` says both are builtins in both shells.